packages/node/test/trailing-slash.test.js

```diff
@@ -81,6 +81,16 @@ describe('Trailing slash', () => {
 				assert.equal(res.status, 200);
 				assert.equal(css, 'h1 { color: red; }\n');
 			});
+
+			it('Does not add trailing slash to astro actions', async () => {
+				const res = await fetch(`http://${server.host}:${server.port}/some-base/_actions/test`, {
+					method: 'POST',
+				});
```

Comment:
I am not sure I understand the test. The current use case is built to have `trailingSlash: always`, this means that calling routes **without slash** should be forbidden, hence return a `404` 🤔

Reply:
Probably yes. Not sure why, but I thought we had fixed the trailing slash issue.

Reply:
Then i'll close this pr and gonna work with `action` implementation

> Probably yes. Not sure why, but I thought we had fixed the trailing slash issue.

Maybe because of this [pr](https://github.com/withastro/astro/pull/12280)